src/components/modals/FeedbackModal/FeedbackModal.tsx

```diff
@@ -0,0 +1,29 @@
+import FeedbackForm from 'src/components/forms/FeedbackForm';
+
+import { Modal } from '../Modal';
```

Comment:
cut on fedbackForm import path
components/modals/Modal is longer than just dots, but more clear